client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -125,13 +159,19 @@ const get_default_state_from_props = (props: _DisplayTableProps) => {
     .mapValues(() => "")
     .value();
 
+  const dropdown_filter = get_default_dropdown_filter(
+    col_configs_with_defaults,
+    data
+  );
+
   return {
     visible_col_keys,
     sort_by,
     descending: false,
     searches,
     initial_props: props,
     page_size: props.page_size_increment,
+    dropdown_filter,
```

Comment:
Minor naming point, but I'd assumed `dropdown_filter` was a function. Something like `filter_options_by_column` would be more descriptive.